### PR TITLE
Added tests for views and utils

### PR DESCRIPTION
**Pull request by kula1922**
- Added tests for views
- Added tests for utils

bugzilla: https://bugzilla.stxnext.pl/show_bug.cgi?id=5948

### CODE REVIEW

File: src/presence_analyzer/tests.py [hunk 1]
```diff
@@ -53,6 +53,59 @@ def test_api_users(self):
         self.assertEqual(len(data), 2)
         self.assertDictEqual(data[0], {u'user_id': 10, u'name': u'User 10'})
 
+    def test_mean_time_weekday_view_when_there_is_bad_user_id(self):
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: You don't have to create function for this case. Move it to ```test_mean_time_weekday_view```

File: src/presence_analyzer/tests.py [hunk 2]
```diff
@@ -53,6 +53,59 @@ def test_api_users(self):
...
+    def test_mean_time_weekday_view(self):
+        """
+        Test mean time weekday view.
+        """
+        resp = self.client.get('/api/v1/mean_time_weekday/10')
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(
+            json.loads(resp.data),
+            [
+                [u'Mon', 0],
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Unicode, use ```from __future__ import unicode_literals```

File: src/presence_analyzer/tests.py [hunk 3]
```diff
@@ -86,6 +139,71 @@ def test_get_data(self):
...
+                    year=2014,
+                    month=1,
+                    day=1,
+                    hour=0,
+                    minute=0,
+                    second=1
+                )
+            ),
+            1
+        )
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Please add another case.

File: src/presence_analyzer/tests.py [hunk 4]
```diff
@@ -86,6 +139,71 @@ def test_get_data(self):
...
+        """
+        Test calculates arithmetic mean when length of items is equal 0.
+        """
+        self.assertEqual(utils.mean([]), 0)
+
+    def test_mean(self):
+        """
+        Test calculates arithmetic mean.
+        """
+        self.assertEqual(utils.mean([1, 2]), 1.5)
```
Comment (dpolcyn) [inline src/presence_analyzer/tests.py]: Please add another case.

File: src/presence_analyzer/tests.py (hunk 3 above)
Comment (kula1922) [inline src/presence_analyzer/tests.py]: Function mean contains complexity mccabe = 1 so this means there is one possible linearly independent paths for this function and there is created test case for it. What another case? could you give me any example?

File: src/presence_analyzer/tests.py (hunk 4 above)
Comment (kula1922) [inline src/presence_analyzer/tests.py]: dito

**Pull request merged**